lib/rules/no-dom-import.ts

```diff
@@ -49,8 +49,10 @@ export default createTestingLibraryRule<Options, MessageIds>({
       if (framework) {
         // marko TL is called @marko/testing-library
         const correctModuleName =
-          framework === 'marko'
-            ? moduleName.replace('dom-', `@${framework}/`)
+          framework === 'angular'
```

Comment:
I'm not sure why this is needed though 😅

Reply:
Because `dom-testing-library` would be replaced with `dom-testing-library` otherwise, which never existed.

Reply:
I don't get it. Is this an issue only for Angular TL? If so, why?

Reply:
Ah, I bet you meant _Because dom-testing-library would be replaced with **angular**-testing-library_

Reply:
Can you add a comment explaining this? Also, the Marko comment is not in the right place now.